src/Test/ProxyTestCase.php

```diff
@@ -125,6 +131,40 @@ protected function getHostName()
     }
 
     /**
+     * Create a request
+     *
+     * @param string $method
+     * @param string $uri
+     * @param array  $headers
+     *
+     * @return RequestInterface
+     * @throws \Exception
+     */
+    protected function createRequest($method, $uri, $headers)
+    {
+        $uri = UriFactoryDiscovery::find()->createUri($uri);
```

Comment:
should we move this to a protected method so it can be customized? just the discovery call i mean.